src/controllers/blocks/BlocksController.ts

```diff
@@ -233,4 +236,72 @@ export default class BlocksController extends AbstractController<BlocksService>
 			await this.service.fetchBlockHeader(hash)
 		);
 	};
+
+	/**
+	 * Return a collection of blocks, given a range.
+	 *
+	 * @param req Express Request
+	 * @param res Express Response
+	 */
+	private getBlocks: RequestHandler<
+		unknown,
+		unknown,
+		unknown,
+		IRangeQueryParam
+	> = async (
+		{ query: { range, eventDocs, extrinsicDocs } },
+		res
+	): Promise<void> => {
+		if (!range) throw new BadRequest('range query parameter must be inputted.');
+
+		// We set a max range to 500 blocks.
+		const rangeOfNums = this.parseRangeOfNumbersOrThrow(range, 500);
+		const rangeOfNumsToHash = await Promise.all(
+			rangeOfNums.map(async (n) => await this.getHashForBlock(n.toString()))
```

Comment:
```suggestion
			rangeOfNums.map((n) => this.getHashForBlock(n.toString()))
```

I guess this is identical anyway?

Reply:
yea this is correct we dont need the async await